c7n/resources/cloudfront.py

```diff
@@ -271,6 +272,29 @@ def process(self, resources, event=None):
         return results
 
 
+@Distribution.action_registry.register('post-finding')
+class DistributionPostFinding(PostFinding):
+
+    resource_type = 'AwsCloudFrontDistribution'
+
+    def format_resource(self, r):
+        envelope, payload = self.format_envelope(r)
+        origins = r['DistributionConfig']['Origins']
+
+        payload.update(self.filter_empty({
+            'DomainName': r['DomainName'],
+            'WebACLId': r.get('WebACLId'),
+            'LastModifiedTime': r['LastModifiedTime'],
+            'Status': r['Status'],
+            'Logging': self.filter_empty(r['DistributionConfig'].get('Logging', {})),
+            'Origins': [
+                dict(Id=o['Id'], OriginPath=o['OriginPath'], DomainName=o['DomainName'])
```

Comment:
Would it make sense to use literal syntax here?
`[{'Id':o['Id'], 'OriginPath':o['OriginPath'], 'DomainName':o['DomainName']} for o in origins]`

Reply:
using the dict as a factory avoids having to quote keys here.